src/tcgwars/logic/impl/gen3/LegendMaker.groovy

```diff
@@ -1191,7 +1191,7 @@ public enum LegendMaker implements LogicCardInfo {
           }
           onAttack {
             def maxSpace = Math.min(my.bench.freeBenchCount, 2)
-            my.deck.search(min:0, max:maxSpace, "Search your deck for up to 2 cards named Omanyte, Kabuto, Aerodactyl, Lileep, or Anorith", {
+            my.deck.search(min:0, max:maxSpace, "Search your deck for ${maxSpace == 2 ? "up to 2 cards" : "a card"} named Omanyte, Kabuto, Aerodactyl, Lileep, or Anorith", {
```

Comment:
I still don't like doing this kind of stuff as it really isn't conducive to adding more languages in the future.

Reply:
Whenever that happens, the entire gstring needs to be translated regardless. As long as the logic is properly converted, it shouldn't be a problem right? It can always be turned down into "up to 1 cards" if needed.

Reply:
Perhaps. Though, let me tell you, plurals are ... "fun" https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals